Scalar.Common/Maintenance/ConfigStep.cs

```diff
@@ -242,10 +242,13 @@ private void ConfigureWatchmanIntegration()
                     this.Context.Enlistment.WorkingDirectoryRoot,
                     ScalarConstants.DotGit.Hooks.QueryWatchmanPath);
 
-                this.Context.FileSystem.CopyFile(
-                    fsMonitorWatchmanSampleHookPath,
-                    queryWatchmanPath,
-                    overwrite: false);
+                if (!this.Context.FileSystem.FileExists(queryWatchmanPath))
+                {
+                    this.Context.FileSystem.CopyFile(
+                        fsMonitorWatchmanSampleHookPath,
+                        queryWatchmanPath,
+                        overwrite: false);
```

Comment:
Did you try setting `overwrite` to `true`?  Would that fix this as well or could that get an access denied if the hook is currently running?

Reply:
That sounds so much simpler.  I just completely missed that.